### PR TITLE
Moviestore landing page

### PR DESCRIPTION
**Pull request by HarshadTupe**

### CODE REVIEW

File: Tupe_Harsh/IMT LANDING/css/style.css [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+    border-radius: 1rem; 
+    border: none;
+    align-items: center;
+    /* padding-top: 2rem; */
+} 
+.main-first button img{
+    height: 5vh;
+}
+/* .main-first button p{
+    padding-top: .6rem;
```
Comment (tauphik257) [inline Tupe_Harsh/IMT LANDING/css/style.css:62]: No commented code pls

File: Tupe_Harsh/IMT LANDING/css/style.css [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+    bottom: 3rem;
+    background-image: url(img/md_5a9797d083160.png);
+    background-size: 100% 100%;
+   
+    /* background-image: url(img/Layer\ 61.jpg);   
+    background-size: 100% 100%; */
+
+}
+/*...................... second portfolio start here .............................*/
+.main-portfolio .portfolio2{
```
Comment (tauphik257) [inline Tupe_Harsh/IMT LANDING/css/style.css:329]: use some meaningful class name avoid `classname1, className2` kind of.

File: Tupe_Harsh/IMT LANDING/css/style.css [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+}
+.main-portfolio .portfolio2 .icon1{
+    /* border: 2px solid black; */
+    height: 6vh;
+    border-radius: 50%;
+    width: 20%;
+    position: relative;
+    left: 9rem;
+    bottom: 3rem;
+    background-image: url(img/md_5a9797d083160.png);
```
Comment (tauphik257) [inline Tupe_Harsh/IMT LANDING/css/style.css:359]: Image name should be proper

File: Tupe_Harsh/IMT LANDING/css/style.css [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+    padding-left: 5rem;
+}
+.main-portfolio .portfolio3 .div3{
+    /* border: 2px solid blue; */
+    height: 20vh;
+    margin-top: 1rem;
+    border-radius: 50%;
+    width: 60%;
+    margin-left: 2.5rem;
+    background-image: url(img/Layer\ 61.jpg);
```
Comment (tauphik257) [inline Tupe_Harsh/IMT LANDING/css/style.css:382]: Image `URL` is not proper

File: Tupe_Harsh/IMT LANDING/img/Layer 46.png
Comment (tauphik257) [inline Tupe_Harsh/IMT LANDING/img/Layer 46.png:1]: Optimize the image size.

File: Tupe_Harsh/IMT LANDING/img/Layer 56.png
Comment (tauphik257) [inline Tupe_Harsh/IMT LANDING/img/Layer 56.png:1]: optimize the image. you can use https://tinypng.com/

File: Tupe_Harsh/IMT LANDING/img/New Project (1).png
Comment (tauphik257) [inline Tupe_Harsh/IMT LANDING/img/New Project (1).png:1]: what is this ?. Please remove if not required

File: Tupe_Harsh/IMT LANDING/index.html [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+        </div>
+        <!-- portfolio -->
+        <section class="main-portfolio">
+            <div class="portfolio1">
+                <div class="div1"></div>
+                <div class="icon"></div>
+                <h3 style="font-size: 15px;color: #0d193d;padding-left:1.5rem;margin-top:-1rem;">Surajit Ghosh Dastidar, PhD </h3>
+                <p style="font-size:12px;color:gray;padding-left:6rem;margin-top:.4rem">CEO</p>
+                <img src="img/actify.png" alt="">
+                <p style="font-size:12px;color:#17205b;text-align:center;padding-right:1rem;padding-left:1rem">Surajit Ghosh Dastidar has done his <br> MBM from Dept. of Business <br> Management, Calcutta University and PhD in Network Optimization. He is <br> a SAS Certified Predictive Modeler. wide range of data science courses and tutorials for a decade</p>
```
Comment (tauphik257) [inline Tupe_Harsh/IMT LANDING/index.html:144]: Remove inline css from the entire page

**Review by tauphik257 — CHANGES_REQUESTED** [review]

File: Tupe_Harsh/IMT LANDING/css/style.css (hunk 1 above)
Comment (HarshadTupe) [reply Tupe_Harsh/IMT LANDING/css/style.css:62]: ok sir

**Pull request closed without merging**